ratelimit/redis.go

```diff
@@ -77,31 +77,27 @@ func (c *clusterLimitRedis) measureQuery(format, groupFormat string, fail *bool,
 	c.metrics.MeasureSince(key, start)
 }
 
-func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func(bool) {
-	nop := func(bool) {}
+func parentSpan(ctx context.Context) (opentracing.Span, bool) {
```

Comment:
I guess we could check result != nil and drop bool

Reply:
done